README.md

```diff
@@ -245,23 +245,23 @@ pip3 install -r requirements-test.txt
 
 To run all unit tests, use:
 ```
-python3 -m unittest discover -p "test*.py"
+DATAPROFILER_SEED=0 python3 -m unittest discover -p "test*.py"
 ```
 
 To run file of unit tests, use form:
 
 ```
-python3 -m unittest discover -p test_profile_builder.py
+DATAPROFILER_SEED=0 python3 -m unittest discover -p test_profile_builder.py
 ```
 
 To run a file with Pytest use:
 ```
-pytest dataprofiler/tests/data_readers/test_csv_data.py -v
+DATAPROFILER_SEED=0 pytest dataprofiler/tests/data_readers/test_csv_data.py -v
```

Comment:
doesn't this need a `;` separating it?

Reply:
Not necessary, but you can. That technically creates a seperate execution

https://unix.stackexchange.com/questions/159489/is-there-a-difference-between-and-and

Reply:
interesting, when i tested locally, that didnt work, curious why it isn't working on my machine